packages/helm/symphony/azure/metadata.json

```diff
@@ -1,37 +1,37 @@
 {
```

Comment:
No need to keep this file in OSS repo, it is only needed in Azure. We will patch them when generating extension.